src/components/ServiceRequestForm.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import Categories from "./services/categories.json";
+import * as Yup from "yup";
+import ErrorMsg from "./errormsg";
+import FormContainer from './FormContainer';
+
+const getSubCategories = (categories, categoryId) => {
+	var category = categories.find(category => category.id === categoryId);
+	return category ? category.types : [];
+};
+
+const ServiceRequestForm = props => {
+
+	const [categories] = useState(Categories);
+	const [subCategories, setSubCategories] = useState([]);
+
+	const handleServiceRequestChange = (changeEvent, setFieldValue) => {
+		const { value } = changeEvent.currentTarget;
+		const subCategories = getSubCategories(categories, parseInt(value));
+		setFieldValue(changeEvent.currentTarget.name, changeEvent.currentTarget.value)
+		setFieldValue('subRequestType', '')
+		setSubCategories(subCategories);
+	};
+
+
+	return (
+
+		<React.Fragment>
```

Comment:
you don't need `<React.Fragment>`,  `FormContainer` can be your top level element. React.Fragment, see [here](https://reactjs.org/docs/fragments.html) for why.

Reply:
React.Fragment has been removed